### DISCUSSION TITLE
How can I solve the error about "Not a registered object."

### DISCUSSION THREAD

**Discussion by Y-Y-JIn** [idaholab/moose#27771] [Q&A Modules: General]
Hello, I got this error message after running the input file. How can I do now?? please help me to fix it.

A 'CoupledSplitCHInterface' is not a registered object.
If you are trying to find this object in a dynamically linked library, make sure that
the library can be found either in your "Problem/library_path" parameter or in the
MOOSE_LIBRARY_PATH environment variable.

**Comment by Rahim-Habibi** [comment]
See this issue #27623 
hope it will help you.
Rahim

**Reply by GiudGiud** [reply]
Yes make sure to include the right module inn your makefile then recompile

**Comment by HakunanMatatat** [comment]
https://github.com/idaholab/mastodon/issues/448

I have the same problem, all the modules in the makefile are loaded.
